tsconfig-null-safe.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
would it be worth extending the existing tsconfig, and then just adding the flags you wanted to override?

Reply:
Fixed!